bin/upload-binaries

```diff
@@ -8,15 +8,27 @@ GITHUB_USER=${GITHUB_USER:-"${CIRCLE_PROJECT_USERNAME}"}
 GITHUB_REPO=${GITHUB_REPO:-"${CIRCLE_PROJECT_REPONAME}"}
 GITHUB_TAG=${GITHUB_TAG:-"${CIRCLE_TAG}"}
 
+function do_publish() {
+	os=$1
+	arch=$2
+	echo "= Uploading fluxctl_${os}_${arch} to GH release ${GITHUB_TAG}"
+	github-release upload \
+		--user ${GITHUB_USER} \
+		--repo ${GITHUB_REPO} \
+		--tag ${GITHUB_TAG} \
+		--name "fluxctl_${os}_${arch}" \
+		--file "build/fluxctl_${os}_${arch}"
+	echo "* Finished pushing fluxctl_${os}_${arch} for ${GITHUB_TAG}"
+}
+
 for arch in amd64; do
```

Comment:
There is no point in having a loop for just an item, please inline it.

Reply:
done